### ISSUE TITLE
nxos_interfaces doesn't shut/no shut an interfaces (enabled: true/false)

### ISSUE THREAD

**Issue by tweippert** [ansible-collections/cisco.nxos#20]
##### SUMMARY

Using nxos_interfaces to enable/disable a interface/port on a Cisco N56128P doesn't work.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
nxos_interfaces

##### ANSIBLE VERSION

```paste below
ansible 2.9.6
  config file = /home/weippertt/.ansible.cfg
  configured module search path = ['/home/weippertt/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/weippertt/ansible-2.9.6/lib/python3.6/site-packages/ansible
  executable location = /home/weippertt/ansible-2.9.6/bin/ansible
  python version = 3.6.8 (default, Jun 11 2019, 15:15:01) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
```

##### CONFIGURATION

```paste below
HOST_KEY_CHECKING(/home/weippertt/.ansible.cfg) = False
PERSISTENT_COMMAND_TIMEOUT(/home/weippertt/.ansible.cfg) = 300
PERSISTENT_CONNECT_RETRY_TIMEOUT(/home/weippertt/.ansible.cfg) = 300
PERSISTENT_CONNECT_TIMEOUT(/home/weippertt/.ansible.cfg) = 300
```

##### OS / ENVIRONMENT

Ansible running on Red Hat Enterprise Linux Server release 7.6 (Maipo)
Cisco Device is:
cisco Nexus 56128P Chassis ("Nexus 56128P Supervisor") 
  system:    version 7.3(3)N1(1)
  bootflash:///n6000-uk9.7.3.3.N1.1.bin

````
system default switchport
system default switchport shutdown
```` 
##### STEPS TO REPRODUCE

Below PLaybook task should configure a Interface on the switch corresponding to some
variable settings. item.value.pEndabled is set to (true/false).

```yaml
- name: Configure Port Description and Status
  nxos_interfaces:
    config:
      - name: "{{ item.value.pName }}"
        description: "{{ item.value.pDesc }}"
        enabled: "{{ item.value.pEnabled }}"
        mode: layer2
    state: merged
  register: port_general
  notify: save_nxos
```

##### EXPECTED RESULTS

Enable/Disable the Interface (shutdown / no shutdown)

##### ACTUAL RESULTS

Task run doesn't change anything with the setting of enabled to true/false
Below is a configuration to set enable it (no shutdown expected). But after
the run the interface looks like:
````
interface Ethernet1/52
  description SELF-SERVICE-TEST
  switchport access vlan 999
````

```paste below
    "invocation": {
        "module_args": {
            "config": [
                {
                    "description": "SELF-SERVICE-TEST",
                    "duplex": null,
                    "enabled": true,
                    "fabric_forwarding_anycast_gateway": null,
                    "ip_forward": null,
                    "mode": "layer2",
                    "mtu": null,
                    "name": "Ethernet1/52",
                    "speed": null
                }
            ],
            "state": "merged"
        }
    }
```

**Comment by NilashishC** [comment]
@tweippert https://github.com/ansible-collections/cisco.nxos/pull/56 should fix this issue. Could you please confirm? Thanks!

**Comment by tweippert** [comment]
@NilashishC i had time to do some enable/disable tests with the fix to interfaces.py and it seems to fix my issue. So, yes i can confirm enable/disable configuration seems to work with this fix!

Is there a change that this get backported/incorporated in 2.9.X Versions?

**Comment by NilashishC** [comment]
@tweippert thank you for confirming. Yes, this fix will be backported and released with the next 2.9 release.
